source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -320,10 +320,15 @@ class WordDocument(UIADocumentWithTableNavigation,WordDocumentNode,WordDocumentB
 	# Microsoft Word duplicates the full title of the document on this control, which is redundant as it appears in the title of the app itself.
 	name=u""
 
+	noNotificationActivityIds = [

```

Comment:
would you mind renaming this to `supressedActivityIds`? If the term "Notification Activity" is relevant then add that to the comment for this list, something like "Notification Activity Ids that are suppressed to reduce unnecessary verbosity"